### ISSUE TITLE
Upper limits for no3 and nh4 in APSIM crop modelling

### ISSUE THREAD

**Issue by santoshkaini** [APSIMInitiative/APSIMClassic#1917]
Hi,
I am a novice user to APSIM crop modeling and trying to simulate “Continuous wheat” module in APSIM. 
I am wondering if you could suggest me in this issue. 
APSIM Warning Error
                     -------------------
     no3ppm =     1200.000
             exceeds upper limit of    300.0000
     Component name: paddock.Loam_field_1 Nitrogen
                      -------------------
     nh4ppm =     700.0000
             exceeds upper limit of    300.0000
     Component name: paddock.Loam_field_1 Nitrogen
When I reduce these values to 300, this issue is resolved. 
**Are the upper limits for both no3 and nh4 are 300 ppm?**
Thank you.

**Comment by peter-devoil** [comment]
Yes, though as this is a warning message, the simulation continues. Usually these limits are set to "mathematically sensible" values - are you sure you observed N at those levels?

**Comment by santoshkaini** [comment]
Thank you, Peter.
